scalafmt-tests/src/test/scala/org/scalafmt/util/HasTests.scala

```diff
@@ -153,37 +149,6 @@ trait HasTests extends AnyFunSuiteLike with FormatAssertions {
 
   def ignore(t: DiffTest): Boolean = false
 
-  def runTest(run: (DiffTest, Parse[_ <: Tree]) => Unit)(t: DiffTest): Unit = {
```

Comment:
looks like `CanRunTests` has now become what `HasTests` used to be, and `HasTests` has smaller role. is this related to munit change or just useful refactor?

Reply:
MUnit related. It's about `FunSuite` inheritance constraints.